config/manifests/bases/sriov-network-operator_configmap.yaml

```diff
@@ -6,6 +6,7 @@ data:
   Broadcom_bnxt_en_BCM57414_NetXtreme-E: "14e4 16d7 16c1"
   Broadcom_bnxt_en_BCM57508_NetXtreme-E: "14e4 1750 1806"
   Intel_i40e_XXV710: "8086 158a 154c"
+  Intel_i40e_X710_X557_AT_10G: "8086 1589 154c"
```

Comment:
This NIC is not listed as supported upstream. It probably should first be added there and then brought into OpenShift. See https://github.com/k8snetworkplumbingwg/sriov-network-operator/commit/37b62c5cdb0fe4d80c8496a173983c594205bb06 as an example.

Also, officially supporting this NIC in OpenShift probably requires coordination with QE, Docs, PM, etc teams.